### ISSUE TITLE
ComfyUI

### ISSUE THREAD

**Issue by R2008KK** [Juqowel/GPU_For_T5#2]
Is there any way to use this extension in comyui?

**Comment by Juqowel** [comment]
No. You can try [https://github.com/neuratech-ai/ComfyUI-MultiGPU](https://github.com/neuratech-ai/ComfyUI-MultiGPU)
Or [this](https://gist.github.com/city96/30743dfdfe129b331b5676a79c3a8a39) for cpu.
